src/dmd/parse.d

```diff
@@ -6076,10 +6076,26 @@ final class Parser(AST) : Lexer
             }
         case TOK.import_:
             {
-                AST.Dsymbols* imports = parseImport();
-                s = new AST.ImportStatement(loc, imports);
-                if (flags & ParseStatementFlags.scope_)
-                    s = new AST.ScopeStatement(loc, s, token.loc);
+                /* https://issues.dlang.org/show_bug.cgi?id=16088
+                 * At this point it can either be an import expression
+                 * or an import declaration. See if the next token after
+                 * `import` is a `(`; if so, then it is an import expression
+                 * and `parseExpression` is called, otherwise it's a
```

Comment:
`an` not `a`

Reply:
Remove "and `parseExpression` is called" - it is redundant as the code itself is pretty obvious on that point.